src/obb_rules/auto_deploys/firingsquad.cljc

```diff
@@ -28,15 +28,27 @@
   (let [lineup (common/build-lineup board stash)
         lineup-quantities (common/build-lineup-quantities lineup stash)
         raw-actions (common/build-deploy-actions player lineup-quantities 7)
-        actions (map deploy/build-deploy raw-actions)
+        actions (map deploy/deploy-action raw-actions)
+        board (assoc board :deploy-actions raw-actions)
         start-result (result/action-success board 0)]
     (reduce (partial common/do-actions player) start-result actions)))
 
+(defn build-history
+  "Takes the auto deploy action and adds the actual run deploy actions
+  on the history"
+  [result raw-actions]
+  (let[history (get-in result [:board :history])
```

Comment:
Yes, the space is missing but the code works. I don't have specific tests for that, but every auto deploy runs that, so it's "covered". I will add specific testes for history though, it makes sense.